equational_theories/MagmaOp.lean

```diff
@@ -44,27 +44,27 @@ theorem models.Op {α} {G : Type} [Magma G] {w₁ w₂ : FreeMagma α} (h : G 
 
 namespace Law.MagmaLaw
 
-def op {α} (l : MagmaLaw α) : MagmaLaw α := { lhs := l.lhs.op, rhs := l.rhs.op}
+def dual {α} (l : MagmaLaw α) : MagmaLaw α := { lhs := l.lhs.op, rhs := l.rhs.op}
```

Comment:
I think these should align (i.e. the `op`s on `l.lhs` and `l.rhs`, `FreeMagma`s should alse become `dual`s

Reply:
Agree. Will do it later. Thanks for the comment!

Reply:
@goens Done, thanks. Can you review it again now?